server/statistics/hot_regions_stat.go

```diff
@@ -13,9 +13,25 @@
 
 package statistics
 
+import (
+	"time"
+)
+
 // HotPeersStat records all hot regions statistics
 type HotPeersStat struct {
-	TotalLoads []float64     `json:"total_loads"`
-	Count      int           `json:"regions_count"`
-	Stats      []HotPeerStat `json:"statistics"`
+	TotalLoads     []float64         `json:"-"`
+	TotalBytesRate float64           `json:"total_flow_bytes"`
+	TotalKeysRate  float64           `json:"total_flow_keys"`
+	Count          int               `json:"regions_count"`
+	Stats          []HotPeerStatShow `json:"statistics"`
+}
+
+type HotPeerStatShow struct {
+	StoreID        uint64    `json:"store_id"`
+	RegionID       uint64    `json:"region_id"`
+	HotDegree      int       `json:"hot_degree"`
+	ByteRate       float64   `json:"flow_bytes"`
+	KeyRate        float64   `json:"flow_keys"`
+	AntiCount      int       `json:"anti_count"`
```

Comment:
I don't think users can understand it.

Reply:
But I think it's helpful for us to debug.